### PR TITLE
refactor: update coding standards

### LINKED ISSUE #2210: update coding standards to lit

### PR DESCRIPTION
**Pull request by dauriamarco**
Closes #2210

### CODE REVIEW

**Review by kyubisation — APPROVED** [review]
LGTM 👍

File: CODING_STANDARDS.md [hunk 1]
```diff
@@ -224,6 +232,28 @@ Boolean properties and return values should use "Whether..." as opposed to "True
 disabled: boolean = false;
 ```
 
+#### Properties initialization
+
+Boolean properties with a default value can be initialized with `false` but not with `true`. This is due to how Lit handles boolean attributes in the DOM: it evaluates, for example, `sanitize="false"` as `true` and not `false` as we would expect. Therefore the property shold be converted in something like `noSanitize = false`:
```
Comment (dauriamarco) [inline CODING_STANDARDS.md:237]:
```suggestion
Boolean properties with a default value can be initialized with `false` but not with `true`. This is due to how Lit handles boolean attributes in the DOM: it evaluates, for example, `sanitize="false"` as `true` and not `false` as we would expect. Therefore the property should be converted in something like `noSanitize = false`:
```

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**